docs/Team/Operations.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Operations"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# The Day to Day
+
+## Resources
+As a company, we have a tendency to throw people in the deep end and see if they can swim. Our business is fast-paced so the traits that allow someone to thrive in that kind of situation are necessary, but we can at least take the alligators out of the deep-end.
+
+### Where Stuff Goes and Where to Find It Again
```

Comment:
This is, IMO, one of the biggest problems. We do have a ton of documentation but unless you know it exists, there's no way to find it. Unless told, for instance, I would have never known to go to recruiting --> root --> blah blah --> blah blah blah to find _client_folders_ in GDrive. If we bubble it more to the surface AND make giving permissions to new people part of an internal onboarding checklist, I think that would eliminate some of the "whoa, how come I've never seen this before?"